sdk/network/Azure.ResourceManager.Network/src/autorest.md

```diff
@@ -229,10 +230,22 @@ directive:
       $.VerificationIPFlowParameters.properties.targetNicResourceId['x-ms-format'] = 'arm-id';
       $.NextHopParameters.properties.targetNicResourceId['x-ms-format'] = 'arm-id';
       $.NextHopResult.properties.routeTableId['x-ms-format'] = 'arm-id';
+      $.Direction['x-ms-enum'].name = 'TrafficDirection';
+      $.ConnectivityIssue.properties.origin['x-ms-enum'].name = 'ConnectivityIssueOrigin';
+      $.ConnectivityIssue.properties.severity['x-ms-enum'].name = 'ConnectivityIssueSeverity';
+      $.ConnectivityParameters.properties.protocol['x-ms-enum'].name = 'NetworkProtocol';
+      $.PacketCaptureStorageLocation.properties.storageId['x-ms-format'] = 'arm-id';
+      $.TroubleshootingProperties.properties.storageId['x-ms-format'] = 'arm-id';
+      $.FlowLogProperties.properties.storageId['x-ms-format'] = 'arm-id';
+      $.FlowLogPropertiesFormat.properties.storageId['x-ms-format'] = 'arm-id';
```

Comment:
$.ConnectivityIssue.properties.origin['x-ms-enum']['name'] = 'ConnectivityIssueOrigin';
      $.ConnectivityParameters.properties.protocol['x-ms-enum']['name'] = 'NetworkWatcherProtocol';

Reply:
The [description for protocol](https://github.com/Azure/azure-rest-api-specs/blob/889a0f6bf32c19174e190c4a5dcc4c74804732c6/specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkWatcher.json#L3282) is `Network protocol`, so maybe we can use `NetworkProtocol` although it's only used in networkWatcher.

Reply:
> The [description for protocol](https://github.com/Azure/azure-rest-api-specs/blob/889a0f6bf32c19174e190c4a5dcc4c74804732c6/specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkWatcher.json#L3282) is `Network protocol`, so maybe we can use `NetworkProtocol` although it's only used in networkWatcher.

NetworkProtocal is not good for this RP as we have too many protocals

Reply:
I see.